frontend/src/components/pages/AboutPage/Title/Title.jsx

```diff
@@ -0,0 +1,10 @@
+import '../../../../../../styles/Title.css';
```

Comment:
нужно еще везде поменять импорты на 
```suggestion
import './Title.scss';
```